GSoC/2015/Julia-Quantum-:-Framework-for-Solvers.md

```diff
@@ -0,0 +1,183 @@
+# Julia Quantum : Framework for Solvers
+
+### Contact Info
+
+Name: Amit   
+Email: bitsjamadagni@gmail.com   
+Location/Timezone: India, GMT +5:30   
+University: Birla Institute of Technology and Science-Pilani, India     
+
+## Personal Background:
+### Motivation for the project:
+
+I am a pre-final year undergraduate pursuing Masters in Mathematics along with Bachelors in Electronics and Electrical Engineering from BITS-Pilani, India. Being a Masters student in Mathematics I have been exposed to courses like Analysis, Algebra, Topology, Differential Geometry among many others. My interest for physics got me started with Quantum Mechanics, and this interest intensified as I started to realize the abstract nature of math being applied to develop axiomatic physical theories. Working with large software libraries always inspired me to construct one from ground zero. I was in search of a project where I could put my efforts in this direction, and I came across Julia. As I searched further I was excited to see Julia Quantum package which had intersections with my interests and since then have tried to contribute to it.
+
+### Programming Skills:
+
+My journey with open source started with SymPy where I had fixed some bugs in Quantum physics module, my interest in Topological Quantum Computation introduced me to the low dimensional topological based ideas of Links/Knots which further led to a Google Summer of Code 2014 project for Sage Mathematical Software where I was involved in developing Computational Knot theory library. The project was based on python and it mainly dealt with conversions between various representations of Links/Knots, on the top of it there were few invariants implemented to distinguish between various Links/Knots. In addition my interest towards hardware allowed me undertake projects involving Raspberry Pi, the details of which could be found on my GitHub repo (Repo titles : [PiSMS : sending SMS using Pi] (https://github.com/amitjamadagni/PiSMS), [Project : Controlling GPIO pins via wireless communication techniques] (https://github.com/amitjamadagni/Project)). I had an opportunity to contribute to Julia and Julia Quantum in the form of enhancing few features. This project would further enhance my knowledge in various domains ranging from Quantum Mechanics to further understanding of Julia. 
+ 
+* GitHub Profile : https://github.com/amitjamadagni
+* GSoC 2014 : https://www.google-melange.com/gsoc/project/details/google/gsoc2014/amitjamadagni/5676830073815040
+* GSoC 2014 blog : https://knotsknotted.wordpress.com/category/gsoc/
+* Contribution to Julia : Implementation of ror! and rol! (https://github.com/JuliaLang/julia/pull/9822)         
+* Contribution to Julia Quantum : Normalize to QuArray (https://github.com/JuliaQuantum/QuBase.jl/pull/15)           
+
+---
+### Project Synopsis : The main inspiration is to equip the library with solvers which are on par with QuTiP. 
```

Comment:
I would put the "main inspiration as a first sentence in this section and extend it slightly. How about

The aim of the project is to equip the JuliaQuantum libraries with solvers which are on par with QuTiP. To this end a flexible and unified framework will be created, which allows to access different solvers. The project will make use of existing packages (if possible) and aims to contribute back in order to homogenize their interfaces and improve the performance.

Main goals of the project include :